app/services/ping.rb

```diff
@@ -0,0 +1,67 @@
+class Ping
+  class << self
+    attr_accessor :extensions
+
+    def ping(request = nil)
+      {
+        'Foreman': {
+          database: ping_database,
+          version: SETTINGS[:version],
+          plugins: Foreman::Plugin.all,
+          smart_proxies: ping_smart_proxies(request),
+          compute_resources: ping_compute_resources(request)
+        }
+      }.merge(ping_extensions)
+    end
+
+    # Could be extended with any class which has ::ping method which returns a hash
```

Comment:
Can you please store this in [Foreman::Plugin](https://github.com/theforeman/foreman/blob/78bc9a43dc811c57b1c3d701cb508748ca935317/app/registries/foreman/plugin.rb)?
Please see https://projects.theforeman.org/projects/foreman/wiki/Registry_pattern for reasoning.